geode-assembly/src/acceptanceTest/java/org/apache/geode/metrics/RegionEntriesGaugeTest.java

```diff
@@ -0,0 +1,380 @@
+    for (String key : keys) {
+      regionOfInterest.put(key, key);
+    }
+    regionOfInterest.destroy(keys.get(0));
+    int expectedNumberOfEntries = keys.size() - 1;
+
+    String getGaugeValueCommand = memberRegionEntryGaugeValueCommand(regionName);
+
+    await().untilAsserted(() -> {
+      GfshExecution execution = gfshRule.execute(connectToLocatorCommand, getGaugeValueCommand);
+      List<Integer> entryCounts = linesOf(execution.getOutputText())
+          .filter(s -> s.startsWith("server"))
+          .map(RegionEntriesGaugeTest::extractEntryCount)
+          .collect(toList());
+
+      assertThat(entryCounts)
+          .as("Number of entries reported by each server")
+          .allMatch(i -> i == expectedNumberOfEntries, "equals " + expectedNumberOfEntries);
+    });
+  }
+
+  @Test
+  public void regionEntriesGaugeShowsCountOfPartitionedRegionValuesInEachServer() {
+    String regionName = "partitionedRegion";
+
+    Region<String, String> region = createRegion(PARTITION.name(), regionName);
+
+    List<String> keys = asList("a", "b", "c", "d", "e", "f", "g", "h");
+    for (String key : keys) {
+      region.put(key, key);
+    }
+
+    region.destroy(keys.get(0));
+
+    int expectedNumberOfEntries = keys.size() - 1;
+
+    String getGaugeValueCommand = memberRegionEntryGaugeValueCommand(regionName);
+
+    await().untilAsserted(() -> {
+      GfshExecution execution = gfshRule.execute(connectToLocatorCommand, getGaugeValueCommand);
+      int sum = linesOf(execution.getOutputText())
+          .filter(s -> s.startsWith("server"))
+          .mapToInt(RegionEntriesGaugeTest::extractEntryCount)
+          .sum();
+
+      assertThat(sum)
+          .as("total number of entries on all servers")
+          .isEqualTo(expectedNumberOfEntries);
+    });
+  }
+
+  @Test
+  public void regionEntriesGaugeShowsCountOfPartitionedRedundantRegionValuesInEachServer()
+      throws IOException {
+    int numberOfRedundantCopies = 1;
+    String regionName = "partitionedRegion";
+
+    int server3Port = AvailablePortHelper.getRandomAvailableTCPPort();
+
+    File folderForServer3 = temporaryFolder.newFolder("server3");
+
+    String startServer3Command = startServerCommand("server3", server3Port, folderForServer3);
+    gfshRule.execute(connectToLocatorCommand, startServer3Command);
+
+    Region<String, String> region =
+        createPartitionedRegionWithRedundancy(regionName, numberOfRedundantCopies);
+
+    List<String> keys = asList("a", "b", "c", "d", "e", "f", "g", "h");
+    for (String key : keys) {
+      region.put(key, key);
+    }
+
+    region.destroy(keys.get(0));
+
+    String getGaugeValueCommand = memberRegionEntryGaugeValueCommand(regionName);
+
+    int numberOfEntries = keys.size() - 1;
+    int totalNumberOfCopies = numberOfRedundantCopies + 1;
+    int expectedNumberOfEntries = numberOfEntries * totalNumberOfCopies;
+
+    await()
+        .atMost(10, TimeUnit.SECONDS)
```

Comment:
I think we've been trying to keep the default timeout, but this seems alright since the `gfsh` command should be quick

Reply:
We probably missed removing this 10 SECOND timeout. We should use the default.

Reply:
just edited the file directly in github since it was such a small change